autograd/numpy/fft.py

```diff
@@ -1,4 +1,5 @@
 from __future__ import absolute_import
+import numpy as np
```

Comment:
See line 33, and line 49.  I had the impression some of the functions are not available in anp (e.g. the inplace assignments). It didn't matter though because we are just creating a constant.